src/test/java/de/qabel/desktop/ui/accounting/AccountingControllerTest.java

```diff
@@ -13,6 +23,17 @@
 
 public class AccountingControllerTest extends AbstractControllerTest {
 
+	String json = "{\"alias\":\"My Name\",\"keys\":{\"curve25519\":{},\"privateKey\":[10,81,75,-68,-37,13,112,-106,-91,97,6,57,50,126,83,-67,-18,-41,-122,-6,60,-114,32,-83,60,106,117,-36,-107,27,114,-11],\"pubKey\":{\"pubKey\":[72,3,-55,4,58,-122,89,-21,-76,-34,36,-37,-78,30,-122,-105,3,95,31,118,-79,-82,-49,119,9,-96,-47,121,94,-77,5,120]}},\"dropUrls\":[{\"uri\":\"http://localhost:5000/OxIA_fJULczG792cZJ2kE66E_OWa45HhlU-RW2GzfAA\"}],\"moduleSettings\":[],\"id\":0,\"created\":1452602138537,\"updated\":0,\"deleted\":0,\"persistenceID\":\"66a36cd4-2019-4fea-95a2-72c9c45e52cd\"}\n";
```

Comment:
if the fixture is not readable (this one isn't, at least because its too long and the used properties doesn't come first) please save it to a (formatted) file